src/dataviews/histogram-dataview/histogram-data-model.js

```diff
@@ -32,6 +33,7 @@ module.exports = Model.extend({
   },
 
   initialize: function () {
+    this.sync = BackboneCancelSync.bind(this);
```

Comment:
I didn't realise this is not the dataview model, but a different model. Plus, [it seems like all DataviewModels are already using `backbone-abort-sync`](https://github.com/CartoDB/cartodb.js/blob/v4/src/dataviews/dataview-model-base.js#L89).

So, let's leave it as is 👍